### ISSUE TITLE
Bug: Default volume set to 50%

### ISSUE THREAD

**Issue by Jdiesel87** [advplyr/audiobookshelf#118]
This might be another personal preference but I like to use Youtube or Netflix as the standard for UI. Both examples default to 100% volume. I think this is the preferred behavior as the device will have global volume controls. In app volume control can be used to balance levels as needed.

**Comment by advplyr** [comment]
Agree, I updated the web player and will update the mobile player on next release.
